R/add_phylopic.r

```diff
@@ -34,13 +37,17 @@
 #'   should be plotted at once. In this case, any other arguments (except for
 #'   `remove_background`) may also be vectors of values, which will be recycled
 #'   as necessary.
+#'   
+#'   If only `color` is specified (and `fill` is NA), the outline and fill color
+#'   will be the same.
 #'
 #'   When specifying a horizontal and/or vertical flip **and** a rotation, the
 #'   flip(s) will always occur first. If you would like to customize this
 #'   behavior, you can flip and/or rotate the image within your own workflow
 #'   using [flip_phylopic()] and [rotate_phylopic()].
 #'
 #'   Note that png array objects can only be rotated by multiples of 90 degrees.
+#'   Also, outline colors do not currently work for png array objects.
```

Comment:
Obviously good to include this here, but do we want to message the user when they provide a raster image and define color and fill? It might save a few people from banging their hands against the wall when the outline is not being added.